src/core/layout/navbar/desktop/index.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { Hidden } from '@material-ui/core'
+
+import { Login } from 'login'
+import { AppBar } from 'core/layout/navbar/appbar'
+import { Header } from 'core/layout/navbar/header'
+
+export const DesktopNavbar = () => <Hidden xsDown>
```

Comment:
![image](https://user-images.githubusercontent.com/23569938/128684843-75bcb0c5-bd29-46a3-a7c5-93933e352767.png)
Since this file doesn't require more ...stuff, we can remove the directory.

Reply:
Done